### ISSUE TITLE
Shipments should accept a list of sending and receiving hubs

### ISSUE THREAD

**Issue by deammer** [distributeaid/shipment-tracker#242]
Notes from a meeting with Dom and Elliot:
- we usually send from and to multiple hubs, so need a multi-select on the Shipment form
- Libby, Dom, and Elliot have added all our hubs to the staging database

In practical terms, we need to update the schema like this:
```graphql
# Before
type Shipment {
  sendingHubId: Int!
  sendingHub: Group!
  receivingHubId: Int!
  receivingHub: Group!
}

# After
type Shipment {
  sendingHubIds: [Int!]!
  sendingHubs: [Group!]!
  receivingHubIds: [Int!]!
  receivingHubs: [Group!]!
}
```

**Comment by deammer** [comment]
I would love some help with the backend/database side of this. I can handle the frontend though.

**Comment by coderbyheart** [comment]
Closed via #274
